juriscraper/opinions/united_states/state/lactapp_2.py

```diff
@@ -0,0 +1,147 @@
+from datetime import datetime
+
+from juriscraper.AbstractSite import logger
+from juriscraper.lib.html_utils import (
+    get_row_column_links,
+    get_row_column_text,
+)
+from juriscraper.OpinionSiteLinear import OpinionSiteLinear
+
+
+class Site(OpinionSiteLinear):
+    first_opinion_date = datetime(2019, 7, 17)
+    days_interval = 28  # Monthly interval
+    abbreviation_to_lower_court = {
+        "Caddo": "First Judicial District Court for the Parish of Caddo, Louisiana",
+        "Ouachita": "Fourth Judicial District Court for the Parish of Ouachita, Louisiana",
+        "Bossier": "Twenty-Sixth Judicial District Court for the Parish of Bossier, Louisiana",
+        "DeSoto": "Forty-Second Judicial District Court for the Parish of DeSoto, Louisiana",
+        "Lincoln": "Third Judicial District Court for the Parish of Lincoln, Louisiana",
+        "Webster": "Twenty-Sixth Judicial District Court for the Parish of Webster, Louisiana",
+        "Franklin": "Fifth Judicial District Court for the Parish of Franklin, Louisiana",
+        "Richland": "Fifth Judicial District Court for the Parish of Richland, Louisiana",
+        "Union": "Third Judicial District Court for the Parish of Union, Louisiana",
+        "Winn": "Eighth Judicial District Court for the Parish of Winn, Louisiana",
+        "Morehouse": "Fourth Judicial District Court for the Parish of Morehouse, Louisiana",
+        "Claiborne": "Second Judicial District Court for the Parish of Claiborne, Louisiana",
+        "Ouachita Monroe City Court": "Monroe City Court for the Parish of Ouachita, Louisiana",
```

Comment:
I appreciate you attempting to capture all of these, but this seems to be a fragile approach.  

I would strip this out and use extract_from_text to capture `lower_court_str`

Reply:
Something like this should work ...

```

    def extract_from_text(self, scraped_text):
        match = re.search(r"Appealed from the\s*(.*?\s*),\s*Louisiana", scraped_text, re.DOTALL)
        if match:
            result = match.group(1).replace("\n", " ")
            return {
                "Docket": {
                    "appeal_from_str": result
                },
            }
        return {}
```

it will require a corresponding addition to the extract from text tests